### PR TITLE
update for Bison Wallet v1 release

### PR DESCRIPTION
**Pull request by buck54321**
This is a rough draft of changes needed for the Bison Wallet v1 release.

What is done
- Direct users towards Bison Wallet, which will temporarily be the only wallet with DCRDEX access
- Temporarily hide Decrediton while we work to get v1-compatible release
- App initialization flow updated to match numerous changes since last edited.

What's not done
- Responsive design checks
- Footer assets and links

This image is what it will look like once Decrediton is ready again. Until then, the Decrediton section will be hidden.

image

### CODE REVIEW

File: src/assets/images/get-started3.png
Comment (jholdstock) [inline src/assets/images/get-started3.png:1]: A "New Password" prompt isn't the most compelling choice for the only picture of the UI. Could this stay as a trade window or something a bit more interesting?

File: src/layouts/partials/get-started.html [hunk 1]
```diff
@@ -54,12 +58,19 @@ <h1>How to Get Started</h1>
...
-                            <span class="bold">Tip:</span> Before moving on, retrieve the application
-                            "seed" from the settings page and store it in a safe place. This is used
-                            to restore wallets and accounts created by the DEX application. Click on
-                            the DCRDEX logo to resume setup.
+                            <p>
+                                Start Bison Wallet or Decrediton.
+                                If using Decrediton, navigate to the DCRDEX interface.
+                            </p>
+                            <p>
+                                Set a new "application password."
```
Comment (jholdstock) [inline src/layouts/partials/get-started.html:66]:
```suggestion
                                Set a new "application password".
```

File: src/layouts/partials/get-started.html [hunk 2]
```diff
@@ -92,14 +104,15 @@ <h1>How to Get Started</h1>
...
-                            just to discourage disruptive behavior.<br><br>
-                            The application will inform you how much is required in your new wallet
-                            to register. But deposit as much as you like to your wallet 
-                            since only the required amount will be spent, and the remaining will be
-                            ready for trading. Deposits must confirm before proceeding. Then click
-                            "Register!" to proceed.
+                            Navigate to the settings view to connect to a DCRDEX access access trading markets.
+                            You can connect to a server and view market activity without cost, but if
+                            you want to trade, you'll need to post a bond
+                        <p>
```
Comment (jholdstock) [inline src/layouts/partials/get-started.html:110]: `<p>` tags need fixing here

File: src/layouts/partials/get-started.html [hunk 3]
```diff
@@ -68,18 +79,19 @@ <h1>How to Get Started</h1>
...
                         <div class="accordion-body">
-                            <p>Choose a server and you will be guided through setup of your
-                            wallets.</p>
-                            <p>There are native (built-in to dexc) wallets for both Decred
-                            and Bitcoin. You may also use your own Electrum wallet for Bitcoin
-                            and Litecoin. For DOGE, ZEC, and BCH, use their full node wallets
-                            (e.g. dogecoind, zcashd, etc.).</p>
-                            <p>For the native wallets, just click "Create!".</p>
-                            <p>You can access the Wallets page at any time.</p>
+                            The quick configuration dialog will offer you the
```
Comment (jholdstock) [inline src/layouts/partials/get-started.html:86]: Want to add `<p>` tags to these paragraphs?

File: src/layouts/partials/get-started.html [hunk 4]
```diff
@@ -92,14 +104,15 @@ <h1>How to Get Started</h1>
...
                         <div class="accordion-body">
-                            Once your wallets are synchronized, you will be prompted
-                            to submit a one-time fee before placing orders. This is a small amount
-                            just to discourage disruptive behavior.<br><br>
-                            The application will inform you how much is required in your new wallet
-                            to register. But deposit as much as you like to your wallet 
-                            since only the required amount will be spent, and the remaining will be
-                            ready for trading. Deposits must confirm before proceeding. Then click
-                            "Register!" to proceed.
+                            Navigate to the settings view to connect to a DCRDEX access access trading markets.
```
Comment (jholdstock) [inline src/layouts/partials/get-started.html:107]:
```suggestion
                            Navigate to the settings view to connect to a DCRDEX server to access trading markets.
```

File: src/layouts/partials/get-started.html [hunk 5]
```diff
@@ -108,12 +121,21 @@ <h1>How to Get Started</h1>
...
                                     <div class="hznt"></div><div class="vrt"></div>
                                 </div>
-                                <div>5. Placing Orders</div>
+                                <div>5. Trade</div>
                             </div>
                         </label>
                         <div class="accordion-body">
-                            To place an order, fill out the price, the lot quantity, and submit it.
-                            Stay online and keep everything running until the trade settles.
+                            Once you are bonded and you have the necessary
```
Comment (jholdstock) [inline src/layouts/partials/get-started.html:128]: `<p>` tags?

**Review by jholdstock — CHANGES_REQUESTED** [review]

File: src/assets/images/bw-logo.png
[the remark this answers is not in the document]
Comment (buck54321) [reply src/assets/images/bw-logo.png:1]: I started doing this. What if I told you that even after optimizing and running through **svgcleaner**, the file size of the SVG is larger than the PNG?
image

[the remark this answers is not in the document]
Comment (peterzen) [reply src/assets/images/bw-logo.png:1]: It's a complex vector path, hence the beefy SVG.  `svgcleaner` will remove cruft added by design tools but it won't alter the paths.  The additional 8k in the file size isn't really a noticeable difference and it'll render a sharp image on all resolutions whereas the PNG won't.

File: src/layouts/partials/get-started.html [hunk 6]
```diff
@@ -92,14 +107,17 @@ <h1>How to Get Started</h1>
...
-                            just to discourage disruptive behavior.<br><br>
-                            The application will inform you how much is required in your new wallet
-                            to register. But deposit as much as you like to your wallet 
-                            since only the required amount will be spent, and the remaining will be
-                            ready for trading. Deposits must confirm before proceeding. Then click
-                            "Register!" to proceed.
+                            <p>
+                                Navigate to the settings view and connect to a DCRDEX server to access trading markets.
+                                You can connect to a server and view market activity without cost, but if
+                                you want to trade, you'll need to post a bond
```
Comment (jholdstock) [inline src/layouts/partials/get-started.html:113]:
```suggestion
                                you want to trade, you'll need to post a bond.
```

File: src/layouts/partials/downloads.html [hunk 7]
```diff
@@ -1,8 +1,108 @@
 <section class="download-section" id="downloads">
-    <div class="container">
+    <div class="my-5">
 
-        <h1>DCRDEX is available in Decrediton or as a standalone application</h1>
-        <h2>v0.6 is now released!</h2>
+        {{ $bwlogo := resources.Get "/images/bw-logo.svg" }}
+        <img src="{{ $bwlogo.Permalink }}" class="mt-4">
```
Comment (jholdstock) [inline src/layouts/partials/downloads.html:5]: This needs some css for mobile.

Please add `id="bisonlogo"` to the img tag and the following somewhere in `downloads.scss`:

```css
#bisonlogo {
    width: 100%;
    max-width: 441px;
}
```

File: src/layouts/partials/downloads.html [hunk 8]
```diff
@@ -1,8 +1,108 @@
...
+        <h4 class="mt-4">Bison Wallet is the official DCRDEX-compatible multi-wallet</h4>
```
Comment (jholdstock) [inline src/layouts/partials/downloads.html:6]:
```suggestion
        <p class="mt-4">Bison Wallet is the official DCRDEX-compatible multi-wallet</p>
```

**Review by jholdstock — CHANGES_REQUESTED** [review]
I've done a quick pass for mobile responsiveness, please adjust the following in `get-started.scss`

Screenshot from 2024-08-27 10-51-43

**Review by jholdstock — APPROVED** [review]

**Comment by buck54321** [comment]
Looks like we'll have all the options for mac users.
image

**Comment by buck54321** [comment]
I think we're good.

**Pull request merged**